app/views/layouts/application.html.haml

```diff
@@ -27,7 +27,7 @@
           %li{class: "#{'active' if current_page?(support_path)}"}
             = link_to "Support", support_path
           %li
-            = link_to "Documentation", "https://registers-docs.cloudapps.digital/", target: :blank, rel: "noopener noreferrer"
+            = link_to "Documentation", "https://registers-docs-testing.cloudapps.digital/", target: :blank, rel: "noopener noreferrer"
```

Comment:
I guess this is OK, but ideally we'd push this to the sandbox instance of registers-frontend so we don't have to modify production just for user testing.

Reply:
Agreed. It's a temp solution whilst registers-trial is locked behind http auth.